source/common/config/well_known_names.h

```diff
@@ -82,6 +82,12 @@ class TagNameValues {
   const std::string SSL_CIPHER = "envoy.ssl_cipher";
   // SSL cipher suite
   const std::string SSL_CIPHER_SUITE = "cipher_suite";
```

Comment:
Should we rename this to add `envoy.` prefix to match other things here?

Reply:
I think it's a good idea, but it's a breaking change.

Do you want me to go ahead with the breaking change ?